CMakeLists.txt

```diff
@@ -77,6 +77,8 @@ endif(CMAKE_CXX_COMPILER_ID STREQUAL "Intel")
 # this is necessary to prevent GCC from warning about ARM64 ABI changes in GCC 10.1
 if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
   add_compile_options(-Wno-psabi)
+  # this prevents the array-bounds warning in AMReX_ParticleContainer.H, but we don't want to disable it globally
+  # add_compile_options(-Wno-array-bounds)
```

Comment:
I think we can get rid of this now, since adding a conditional prevents this warning.